src/helpers/terraform.js

```diff
@@ -239,10 +239,17 @@ class Terraform {
     }
 
     const workspace = this._tf.workspace;
-    const regex = new RegExp(`(\\*\\s|\\s.)${workspace}$`, 'm');
+    const regexAll = new RegExp(`(\\*\\s|\\s.)${workspace}$`, 'm');
```

Comment:
@Leichtwind this regexp was suppose to match selected and not selected workspaces from list, so if you check selected with another regexp this one could be simplified

Reply:
@ddimitrioglo Okay, got it. I will improve it tomorrow when I come to office